addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1940,4 +1941,447 @@ options.registry.SnippetMove = options.Class.extend({
         }
     },
 });
+
+options.registry.InnerChart = options.Class.extend({
+    custom_events: _.extend({}, options.Class.prototype.custom_events, {
+        'get_custom_colors': '_onGetCustomColors',
+
+    }),
+    events: _.extend({}, options.Class.prototype.events, {
+        'click we-button.add_column': '_onAddColumnClick',
+        'click we-button.add_row': '_onAddRowClick',
+        'click we-button.o_we_matrix_remove_col': '_onRemoveColumnClick',
+        'click we-button.o_we_matrix_remove_row': '_onRemoveRowClick',
+        'blur we-matrix input': '_onFocusOutMatrixInput',
+        'focus we-matrix input': '_onFocusMatrixInput',
```

Comment:
I don't mind people other convention than me for this, but not in the same widget... above you write `_onXXXClick` and here `_onFocusXXX`. Put the event name at the end. Especially when reading "_onFocusMatrixInput", I imagine an input event into a "focus matrix" component